glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 933723e6eabebd9954e5823b2912f7ef87c7f46a53cc6ffad9ee2624ee5eb85c
-updated: 2019-04-18T12:06:00.343433-04:00
+updated: 2019-04-30T14:24:04.046222-04:00
```

Comment:
Maybe revert `glide.lock` changes?  I see you didn't change `glide.yaml` so this changes to this file is unnecessary I believe.